docs/getting_started/basic_tests.rst

```diff
@@ -33,6 +33,35 @@ gives access to the following parameters:
 - ``case_id``: current test case identification number (optional for test suite setup and teardown)
 - ``aux_list``: list of used auxiliaries
 
+
+``suite_id`` and ``case_id`` are used to clearly define a test execution order but remain optional(still remain
+mandatory for remote testing). If both are not defined (by default ``suite_id`` and ``case_id`` are equal to 0),
```

Comment:
```suggestion
``suite_id`` and ``case_id`` are used to coordinate and define a clear execution order.  It is now optional for ``pykiso.BasicTest`` but still mandatory in case of ``pykiso.RemoteTest``.
If both ids  are not defined (by default ``suite_id`` and ``case_id`` are equal to 0),
```